Messaging/Forward Messages to Email - Node.js/.env.sample

```diff
@@ -0,0 +1,7 @@
+SPACE_URL=<YOUR SPACE URL>
```

Comment:
it seems that this is not needed in the code? Should it be removed from here?

Reply:
you are correct. Removed